scripts/check_test_list.py

```
@@ -1,27 +1,512 @@
+
+    ast_cache = {}
+    errors = []
+
+    for ref in active_malformed + waive_malformed:
+        errors.append(f"MALFORMED ENTRY: {ref}")
+
+    # Flag waive IDs that are a strict prefix of an active ID (likely truncated).
+    waive_only_keys = {k for k in waive_entries if k not in active_entries}
+    for rel_path, class_name, func_name, param_id in sorted(
+            waive_only_keys, key=lambda x:
+        (x[0], x[1] or "", x[2], x[3] or "")):
```

Flake8 (7.3.0)

[error] continuation line unaligned for hanging indent

(E131)